### DISCUSSION TITLE
Why Is "Redirect mobile forums to main forums" An Easter Egg Addon?

### DISCUSSION THREAD

**Discussion by ghost** [ScratchAddons/ScratchAddons#3009] [Help]
When you do the Konami Code, a bunch of addons with an icon of an easter egg appear. One of those is "Redirect mobile forums to main forums". I don't see why that is an easter egg, seems rather helpful.

**Comment by TheColaber** [comment]
its actually pretty useless.

**Reply by mxmou** [reply]
It isn't. When searching for a forum topic on Google, many results are from the mobile forums.

**Reply by WorldLanguages** [reply]
I planned to remove it from easter eggs in #2908
